apps/src/templates/studioHomepages/borderedCallToAction.module.scss

```diff
@@ -0,0 +1,37 @@
+@import 'color';
```

Comment:
I imagine a lot of these are translated from the styles in the old component.. I think the proof is in the screenshots!

Reply:
Correct! I refactored it a bit too to clean things up and make it responsive.